index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Document</title>
+</head>
+<body>
+  <nav class="navbar">
+    <h4 class="name">Oscar Espinoza</h4>
+    <div class="toggle-btn">
+      <span></span>
+      <span></span>
+      <span></span>
+    </div>
+    <div class="nav-links">
+      <p>Portfolio</p>
+      <p>About</p>
+      <p>Contact</p>
+    </div>
+  </nav>
```

Comment:
Nice work creating the nav bar, it looks perfect! 🏆 

- [ ] Here, the logo or in this case your name should be included inside a `header` tag, the nav bar should only include the links of the different sections of your future hamburger button (mobile version). 
Please kindly change your <div class="nav-links"> for a `nav` tag. An wrap it inside the `header` it's applying the best practices of HTML.

It should look like this.

![image](https://user-images.githubusercontent.com/93630700/198167183-0f72267f-d196-47a6-8db6-6ef9b18931f1.png)

- [ ] Please add a display `flex` property to the `header`

- This gives us a lot of options to align all our elements, plus, it's responsive 🟢 🟢

Reply:
Done